arch/x86/Makefile.mmu

```diff
@@ -0,0 +1,29 @@
+ifeq ($(KBUILD_VERBOSE),1)
+	GENMMU_EXTRA_ARGS := -l
+else
+	GENMMU_EXTRA_ARGS :=
+endif
+
+ifeq ($(PREBUILT_HOST_TOOLS),)
```

Comment:
You don't need this. Just point it to scripts/gen_mmu.py.
This is only for C-based build tools.